### PR TITLE
Tests for de1.on() and de1.off()

### PR DESCRIPTION
**Pull request by LukasBombach**
To be honest I am not 100% happy with the tests, but I'd be ok with this for now and I finally want to see de1 go live and not win a trophy for the best code on earth.

\* disclaimer I am super tired and may regret this text later 😬

### CODE REVIEW

File: packages/de1/tests/de1/events.test.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+describe("de1 events", () => {
+  const de1 = new DE1();
+  const emitter = new EventEmitter();
+
+  // todo we don't have enough events
+  // todo this mapping code is horrible
+  const namesWithEventFixture = Object.keys(events)
+    .map(uuid => Object.entries(converters).find(([, c]) => c.uuid === uuid))
+    .map(c => c![0] as Name);
```
Comment (oscar-b) [inline packages/de1/tests/de1/events.test.ts:15]: Funky! :D

**Review by oscar-b — APPROVED** [review]
I think this looks fine! Should we dump some more events and add before merging?

File: packages/de1/tests/de1/events.test.ts (hunk 1 above)
Comment (LukasBombach) [reply packages/de1/tests/de1/events.test.ts:15]: I‘m sorry 🤦🏻‍♂️

Comment (LukasBombach) [reply packages/de1/tests/de1/events.test.ts:15]: Just tired and wanna be done already 🤦🏻‍♂️

**Comment by LukasBombach** [comment]
> I think this looks fine! Should we dump some more events and add before merging?

sorry i did not see that, i will create an issue for this for some time in the future

**Comment by LukasBombach** [comment]
There we go https://github.com/LukasBombach/de1/issues/23

**Pull request merged**